receipt_agent/examples/batch_validate.py

```
@@ -0,0 +1,184 @@
+            ValidationStatus.ERROR: "💥",
+        }
+        emoji = status_emoji.get(result.status, "❓")
+        print(f"  {emoji} {image_id}#{receipt_id}: {result.status.value} ({result.confidence:.0%})")
+
+    # Print summary
+    print("\n" + "=" * 60)
+    print("BATCH VALIDATION SUMMARY")
+    print("=" * 60)
+    print(f"Total receipts: {len(results)}")
+    print(f"Merchant: {merchant_name}")
+    print(f"\nResults by status:")
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)